### ISSUE TITLE
the result in pascal is poor

### ISSUE THREAD

**Issue by makalo** [Engineering-Course/CIHP_PGN#7]
Hi,thanks for your work.When I train PGN in pascal,I get a poor result that the miou is 0.38. besides,In your code,you use 'tf.reduce_sum' to caculate the edge loss,why not 'tf.reduce_mean'.this cause the edge loss is vary large,above 10000.But when I use 'tf.reduce_mean',I get a worse result.Why?
could you explain the reason and share how do you trian pascal.Thank you.

**Comment by Engineering-Course** [comment]
There are many reasons for the poor re-trained results, such as pre-trained model, training epochs.
We use sum operation for edge loss to follow the baseline, 
> Holistically-Nested Edge Detection

**Comment by GuoxiaWang** [comment]
@makalo @Engineering-Course 

I am doing the edge/boundary detection task. Yes, as you see the training loss, edge loss is very large, maybe 10000. It is a normal phenomenon.  Because edge detection model using class-balanced cross entropy loss function.
